src/main/java/frc/robot/commands/ClimbAngleCommand.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.BackClimber;
+
+
+public class ClimbAngleCommand extends CommandBase {
+  private final BackClimber m_climber;
+  private final Rotation2d m_setpoint;
+
+  public ClimbAngleCommand(BackClimber climber, Rotation2d setpoint) {
```

Comment:
We should rename this to BackClimbAngleCommand. 

We should also create a FrontClimbAngleCommand, so we can move the front climb arms to a specified angle. I don't think we need PID for the front, so we could probably go back to this version for a start: https://github.com/Team2470/FRC-2022-robot/blob/e7edc3c911a0c4afca847b528dd9b867eb84be94/src/main/java/frc/robot/commands/ClimbAngleCommand.java